scripts/e2e_test_docker.sh

```diff
@@ -18,6 +18,12 @@ set -o errexit
 set -o nounset
 set -o pipefail
 
+delete_cluster_controller_image() {
+    aws ecr-public batch-delete-image --repository-name eks-anywhere-cluster-controller --image-ids=imageTag=${PULL_PULL_SHA}.${PULL_BASE_REF} --region us-east-1
```

Comment:
So the latest will never be cleaned up, it will always be based on any PR open right?